File: campus-backend-senai-javascript/estudo-backend-senai-site/exercicios-migrados/M03/Exercicio02.js
// Exercicio02.js
// Objetivo: produzir uma tabuada com repetição.

const numero = 7;
let contador = 1;

while (contador <= 10) {
    const resultado = numero * contador;
    console.log(`${numero} x ${contador} = ${resultado}`);
    contador++;
}

File: src/modulo03_lacos_repeticao_while/Exercicio02.java
package modulo03_lacos_repeticao_while;

/*
 * ============================================================
 * Curso Java
 * Módulo 03 - Laços de Repetição
 * Aula 01 - While
 * Exercício 02
 * ============================================================
 *
 * Objetivo:
 * Utilizar a estrutura while para exibir os números pares
 * de 2 a 10, demonstrando o uso de incremento em laços
 * de repetição.
 */

public class Exercicio02
{
    public static void main(String[] args)
    {
        int contador = 2;

        while(contador <= 10)
        {
            System.out.println(contador);
            contador+=2;
        }
    }
}
